### DISCUSSION TITLE
Add phones with barometer sensor and how it works

### DISCUSSION THREAD

**Discussion by rgmf** [OpenTracksApp/OpenTracks#954] [Ideas]
**Long story short:**

Would be a good idea adding smartphones with barometer sensor to the README_TESTED_SENSORS.md file and append an infor describing how they work?

There are smartphones with barometer sensor that work great and other don't.

**Long story:**

I've had the chance of test several smartphones (from friends and mines) and I found out that some of them have barometer sensor but they're useless.

The last one was a Pixel 3 that gave me a gain of 12000m in a 100m gain route. Look into this weird value I found out that Pixel 3 changes its pressure value when you tap on the screen or when you try to bend it, so if you put it on bike handlebar the Pixel 3 is adding gain quickly because the vibrations.

The same happened to me with Xperia XZ1.

In other way, Pixel 2XL or Pixel 3a works like a charm. So seems there are good and bad smartphones for computing the gain using the barometer sensor.

**Comment by dennisguse** [comment]
Sounds like a great idea!
We may also need to add the Android version (original vs custom aka LineageOS).
